templates/ovs/bin/init.sh

```diff
@@ -28,18 +39,6 @@ EnableChassisAsGateway=${EnableChassisAsGateway:-false}
 PhysicalNetworks=${PhysicalNetworks:-""}
 OvnHostName=${OvnHostName:-""}
 
-function wait_for_ovsdb_server {
```

Comment:
nit: is there a reason why the function is moved up? it's not called until after set -ex below anyway.

Reply:
No, I changed that back :)